### PR TITLE
install script major re-org

### PR DESCRIPTION
**Pull request by CleoQc**
Move wiringPi to raspbian for robots and still pull it from here.
Reorg the install script
fix the IR install script due to move out of Desktop

### CODE REVIEW

File: Setup/install.sh [hunk 1]
```diff
@@ -1,210 +1,257 @@
...
-    echo " |_   _|         | |         | |      (_)           ";
-    echo "   | |  _ __   __| |_   _ ___| |_ _ __ _  ___  ___  ";
-    echo "   | | | '_ \ / _\ | | | / __| __| '__| |/ _ \/ __|";
-    echo "  _| |_| | | | (_| | |_| \__ \ |_| |  | |  __/\__ \ ";
-    echo " |_____|_| |_|\__,_|\__,_|___/\__|_|  |_|\___||___/ ";
-    echo "                                                    ";
-    echo "                                                    ";
-    echo " "
-fi
+called_from_di_update() {
```
Comment (DexterInd) [inline Setup/install.sh]: Documentation.

File: Setup/install.sh [hunk 2]
```diff
@@ -1,210 +1,257 @@
...
+    if [ -f update_wiringpi.sh ]
+    then
+        sudo rm update_wiringpi.sh
+    fi
 
-echo " "
-echo "Making I2C changes in /boot/config.txt . . ."
-echo "================================================"
+    # this path needs updating
+    sudo curl https://raw.githubusercontent.com/CleoQc/Raspbian_For_Robots/update201612/upd_script/update_wiringpi.sh | bash
```
Comment (DexterInd) [inline Setup/install.sh]: Should reference DexterInd not CleoQC?

File: Setup/install.sh [hunk 3]
```diff
@@ -1,210 +1,257 @@
...
+    echo " "
+}
+
+install_arduino() {
+    #Adding ARDUINO setup files
+    echo " "
+    echo "Making changes to Arduino . . ."
+    echo "==============================="
+    cd /tmp
+    wget http://project-downloads.drogon.net/gertboard/avrdude_5.10-4_armhf.deb
```
Comment (DexterInd) [inline Setup/install.sh]: Should this use our latest AVRDude install script?

File: Setup/install.sh [hunk 4]
```diff
@@ -1,210 +1,257 @@
...
+    sudo dpkg -i avrdude_5.10-4_armhf.deb
+    sudo chmod 4755 /usr/bin/avrdude
+
+    cd /tmp
+    if [ -f /tmp/setup.sh ]; then
+        rm /tmp/setup.sh
+    fi
+    wget http://project-downloads.drogon.net/gertboard/setup.sh
```
Comment (DexterInd) [inline Setup/install.sh]: Ditto dependency on outside source.

File: Setup/install.sh (hunk 2 above)
Comment (CleoQc) [inline Setup/install.sh]: Waiting for it to have a permanent URL to pull from

**Pull request merged**